diff --git a/scripts/expired_domains.sh b/scripts/expired_domains.sh
index 52c3d83..bd25439 100755
--- a/scripts/expired_domains.sh
+++ b/scripts/expired_domains.sh
@@ -75,7 +75,7 @@ do
         echo  "$ips" >> $TEMPORARY.5
     elif [ $status_code -ne "200" ] && [ $status_code -ne "000" ]
     then
-        echo  "$ips $status_code" >> $MAIN_PATH/expired-domains/$FILTERLIST-unknown.txt
+        echo  "$ips" >> $MAIN_PATH/expired-domains/$FILTERLIST-unknown.txt
     else
         echo "Test"
     fi
@@ -96,7 +96,7 @@ do
     status_code=$(curl -o /dev/null --silent --head --write-out '%{http_code}\n' $ips)
     if [ $status_code -ne "200" ]
     then
-        echo  "$ips $status_code" >> $MAIN_PATH/expired-domains/$FILTERLIST-unknown.txt
+        echo  "$ips" >> $MAIN_PATH/expired-domains/$FILTERLIST-unknown.txt
     else
         echo "Test"
     fi
