diff --git a/scripts/ECODFF.sh b/scripts/ECODFF.sh
index 26e4c9c..32a99be 100755
--- a/scripts/ECODFF.sh
+++ b/scripts/ECODFF.sh
@@ -104,7 +104,7 @@ for i in "$@"; do
         if [ "$status_code" -eq "000" ]; then
             echo "$domain" >> "$TEMPORARY".5
         elif [ "$status_code" -ne "200" ] && [ "$status_code" -ne "000" ]; then
-            echo "$domain $status_code" >> "$MAIN_PATH"/expired-domains/"$FILTERLIST"-unknown.txt
+            echo "$domain" >> "$MAIN_PATH"/expired-domains/"$FILTERLIST"-unknown.txt
         else
             echo "Test"
         fi
@@ -148,7 +148,7 @@ for i in "$@"; do
     while IFS= read -r domain; do
         status_code=$(curl -o /dev/null --silent --head --write-out '%{http_code}\n' "$domain")
         if [ "$status_code" -ne "200" ]; then
-            echo "$domain $status_code" >> "$MAIN_PATH"/expired-domains/"$FILTERLIST"-unknown.txt
+            echo "$domain" >> "$MAIN_PATH"/expired-domains/"$FILTERLIST"-unknown.txt
         else
             echo "Test"
         fi
-- 
2.22.0

